contribs/gmf/src/services/permalink.js

```diff
@@ -453,12 +453,14 @@ gmf.Permalink.prototype.addListenerKey_ = function(uid, key, opt_isol) {
  */
 gmf.Permalink.prototype.getMapCenter = function() {
   let center = null;
-  const x = /** @type {number} */ (this.ngeoStateManager_.getInitialValue(
-    gmf.PermalinkParam.MAP_X));
-  const y = /** @type {number} */ (this.ngeoStateManager_.getInitialValue(
-    gmf.PermalinkParam.MAP_Y));
+  const x = this.ngeoStateManager_.getInitialValue(
```

Comment:
minor but can you put all in the same line?
```js
const x = this.ngeoStateManager_.getInitialValue(gmf.PermalinkParam.MAP_X);
```

same for all the changes in this pr